### ISSUE TITLE
SUPPORTS_EXPRESSION_COMPILE_WITH_PREFER_INTERPRETATION_PARAM

### ISSUE THREAD

**Issue by jonreis** [dadhi/DryIoc#229]
Argument 1: cannot convert from 'bool' to 'System.Runtime.CompilerServices.DebugInfoGenerator'

**Comment by jonreis** [comment]
Sorry hit wrong key

**Comment by dadhi** [comment]
So, is this closed or not?

**Comment by jonreis** [comment]
@dadhi very sorry about this.
I was attempting to use DryIoc as a source code include, however; I got a compilation error:

Severity	Code	Description	Project	File	Line	Suppression State
Error	CS1503	Argument 1: cannot convert from 'bool' to 'System.Runtime.CompilerServices.DebugInfoGenerator'	Connexion.Share	C:\Users\jonre\.nuget\packages\dryioc\4.1.0\contentFiles\cs\net45\DryIoc\Container.cs	3767	N/A

It seems as though Visual Studio 2019 is not properly setting NET461 when 461 is the targeted framework version. To work-around this issue I manually set NET461 in the project build "Conditional compilation symbols". A hack for sure. I am using Visual Studio Version 16.4.5 in for your info. Also, when I used NET472 I did not get this error.

I am pretty sure this is not a bug in DryIoc, rather an issue with Visual Studio. As far as I can tell all the compiler directives are correct.
